inst/stan/linear_single_ind.stan

```diff
@@ -85,27 +85,22 @@ model {
     }
 
     if(i < n_obs){
-      //Estimate growth rate
+      //Estimate next size
       y_hat[i+1] = rk4(y_hat[i], pars, (time[i+1] - time[i]), step_size);
     }
   }
 
   //Likelihood
-  y_obs ~ normal(y_hat, global_error_sigma);
+  y_obs ~ normal(y_hat, 0.1);
```

Comment:
this branch adds a vignette, but there's also some change here to R functions and Stan files. are these intended?